### ISSUE TITLE
Capitalize subtitles that start with interpunction

### ISSUE THREAD

**Issue by rudolf-adamkovic** [andras-simonyi/citeproc-el#143]
BibTeX entry:

```bibtex
@Article{myers-1988,
  author       = {Myers, Norman},
  title        = {Threatened biotas},
  journal      = {Environmentalist},
  year         = 1988,
  volume       = 8,
  number       = 3,
  pages        = {187-208},
  month        = {Sep},
  type         = {Abstract},
  subtitle     = {"Hot spots" in tropical forests},
  day          = 01,
  issn         = {1573-2991},
  doi          = {10.1007/BF02240252}
}
```

Actual APA rendering:

Myers, N. (1988). Threatened biotas: “hot spots” in tropical forests [Abstract]. Environmentalist, 8(3), 187–208. https://doi.org/10.1007/BF02240252

Expected APA rendering:

Myers, N. (1988). Threatened biotas: “Hot spots” in tropical forests [Abstract]. Environmentalist, 8(3), 187–208. https://doi.org/10.1007/BF02240252

[Notice the capital "H" in "Hot spots".]

**Comment by andras-simonyi** [comment]
First of all, apologies for looking into this only now... it should be fixed by PR #163 which I have just merged into the master branch.

**Comment by rudolf-adamkovic** [comment]
Super!  Thank you.
